src/pages/api/thirdparty/resolvers.test.ts

```diff
@@ -0,0 +1,106 @@
+import { ApolloServer } from '@apollo/server'
+import { gql } from 'graphql-tag'
+import KeystoneAPI from './dataSources/thirdPartyKeystone'
+import { resolvers, ThirdPartyContext } from './resolvers'
+import { typeDefs } from './schema'
+import type { SingleGraphQLResponse } from 'types'
+
+let server: ApolloServer<ThirdPartyContext>
+let serverContext: ThirdPartyContext
+let keystoneAPI: KeystoneAPI
+
+describe('GraphQL resolvers', () => {
+  beforeEach(() => {
+    server = new ApolloServer({
+      typeDefs,
+      resolvers,
+    })
+
+    const { cache } = server
+    keystoneAPI = new KeystoneAPI({ cache })
+
+    keystoneAPI.getPublicArticles = jest.fn(async () => {
+      return testCNote
+    })
+
+    serverContext = {
+      dataSources: {
+        keystoneAPI,
+      },
+    }
+  })
+  describe('Query.articles', () => {
+    type ResponseData = {
+      articles: {
+        id: string
+        title: string
+        publishedDate: string
+        body: {
+          document: {
+            type: string
+            children: {
+              text: string
+            }[]
+          }[]
+        }
+      }[]
+    }
+
+    test('returns all articles tagged with C-Note', async () => {
```

Comment:
Suggestion(non-blocking): For a test that checks the negative case, ie they pass something that isn't `CNOTE` and maybe one without. This isn't blocking given we are in the early stage of figuring out what our users will need.